lib/ruby-measurement/definitions/us_customary/volume.rb

```diff
@@ -8,6 +8,8 @@
   unit.convert_to(:'fl oz') { |value| value * 128.0 }
   unit.convert_to(:tbsp) { |value| value * 256.0 }
   unit.convert_to(:tsp) { |value| value * 768.0 }
+  unit.convert_to(:ml) { |value| value * 3785.411784 }
+  unit.convert_to(:l) { |value| value * 3.785411784 }
```

Comment:
This won't work in conjunction with [this require approach](https://github.com/mhuggins/ruby-measurement#specifying-units).  Some users do not want to load all unit definitions, but simply the ones they need.  If someone loads just `us_customary` or `us_customary/volume` definitions, then these definitions won't work.

Perhaps these belong in a separate require, but let me think on it.  It might just make sense to expose an interface for adding custom conversions within your own project as well.